sound/soc/sof/debug.c

```diff
@@ -5,7 +5,8 @@
 //
 // Copyright(c) 2018 Intel Corporation. All rights reserved.
 //
-// Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+// Authors: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+//          Cezary Rojewski <cezary.rojewski@intel.com>
```

Comment:
We dont update author information for modifying a source file. the commit sign-off is good enough.

Reply:
Ack. Used to updating authors one sufficient enough update is provided by given individual.

Reply:
Fixed in v5.